requirements.txt

```diff
@@ -0,0 +1,2 @@
+pynetbox
+packages
```

Comment:
Is there a way to limit this to when Ansible versions < 2.10 is installed? I did a brief look at https://peps.python.org/pep-0508/#environment-markers but don't have experience with it to make a useful suggestion for how to implement it